# Enforce that string literals are in sentence case. (sentence-case)

## Rule Details

Examples of **incorrect** code for this rule:

```js

'A String in Title Case'

```

Examples of **correct** code for this rule:

```js

'A string in sentence case'
'lowercase-string'

```
